index.html

```diff
@@ -1412,6 +1412,153 @@ <h3>Biometrics</h3>
+        
+        <p>
+          The energy usage and other environmental impacts of a DID-based solution
+          should also be compared to existing approaches, which are often paper based
+          and/or require shipment of legacy certificates via air or other methods, and
+          thus can have extremely high environmental impacts. These should be factored
+          in when performing an assessment of energy usage and environmental impacts
+          of potential replacement solutions based on DIDs.
+        </p>
+        
+        <p>
+          Utilizing or authoring DID methods that compromise on
+          the principles of security and / or control for other principles
+          of such as environmental concerns should be very carefully 
+          considered in balance against alternative approaches that may
+          provide the highest levels of security and protection.
+          The specific use case for a DID method will help determine the
+          optimal approach.  For matters that concern human rights, selection 
+          of approaches that are weighted towards privacy and control should 
+          outweigh other considerations.
+        </p>
+
+        <p class="advisement">
+          It is strongly recommended that authors of DID methods provide an assessment
+          of energy usage and impact of their DID method, preferably performed by
+          an independent third party.
+        </p>
+  
+      </section>
+    </section>
+
+    <section class="informative">
+      <h3>Considerations for Applications related to Human Rights</h3>
+      <p>
+        Use of DIDs in furtherance of the improvement of the overall condition of 
+        the human species is highly desirable.  Examples of related work where 
+        decentralized technologies have been utilized to prevent use of minerals
+        sourced from conflict zones may be found in the following reports from
+        <a href="https://sustainability.google/progress/projects/traceability/">Google</a>
+        and
+        <a href="https://www.apple.com/supplier-responsibility/pdf/Apple-Conflict-Minerals-Report.pdf">Apple</a>.
+        DIDs serve a unique purpose in supporting privacy and enabling individual 
+        control over identifiers while also enabing accurate tracing of an individual
+        item's provenance to ensure that that item did not originate from use of
+        forced labor or have other aspects in its production process or supply 
+        chain that were inconsistent with a basic respect for human rights. 
+      </p>
+
+      <p>
+        Use of DIDs to trace the environmental impacts of various supply chains 
+        has already been performed, specifically in areas of food and agricultural
+        products.  
+        This type of usage demonstrates clearly that the new technology can be utilized 
+        in a manner that the consumed energy is balanced by the advantage of being able
```

Comment:
Consumed energy is not an appropriate call out here.

I tried to replace that language with "environmental impact" but then the entire sentence didn't make sense.